pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -283,6 +283,9 @@ func (configgen *ConfigGeneratorImpl) buildSidecarInboundHTTPListenerOptsForPort
 			},
 			ServerName:          EnvoyServerName,
 			DelayedCloseTimeout: features.DelayedCloseTimeout,
+			Http2ProtocolOptions: &core.Http2ProtocolOptions{
```

Comment:
can we just set `Http2ProtocolOptions` on all HTTP listeners like this?

Reply:
I think this will end up conflicting here: https://github.com/istio/istio/blob/master/pilot/pkg/networking/core/v1alpha3/listener.go#L290

As for whether it can be set all of the time - I _think_ so, since the protocol selection is determined by `codec_type`, so I would imagine it doesn't matter for non-h2 traffic (despite the line I linked seeming to imply it is doing something). @lambdai ?